src/app/core/ServiceProvider.php

```diff
@@ -33,32 +31,10 @@ private function bootstrap(): void
         $this->map = [
             UserController::class => function (string $class, ServiceProvider $serviceProvider) {
                 return new $class(
-                    $serviceProvider->make(UserRepository::class),
+                    $serviceProvider->make(RESTRepository::class),
```

Comment:
Should: The name here should not be abstract. Why not RestUserRepository? or even GoRestUserRepository, by service name.